doc/help.md

```diff
@@ -84,48 +219,35 @@ include: package:pedantic/analysis_options.yaml
-The maintenance score reflects how tidy and up-to-date a package is. A package
-starts with 100 points, which is then then subtracted deductions based on
-[a long list of detailed checks][pana-maintenance]. In summary, the main
-components that influence this score are:
-  - Last publish date: Best if the package has been published within one year.
-  - Up-to-date dependencies: Best if all of the package's dependencies are on the latest version.
-  - `README.md`, `CHANGELOG.md`, and example files: Best if all are present. For
-    information on how to provide these files, see the [pub package layout conventions][pub-layout].
-  - `analysis_options.yaml`: Best if this file is present. For more information, see
-    [Customize Static Analysis][analysis-options].
+This category measures if a package has up-to-date dependencies in three areas:
 
-*How can you improve your maintenance score?*
+  - Works with the latest stable Dart SDK.
+  - Works with the latest stable Flutter SDK (if applicable).
+  - Works with the latest versions of all pendendencies.
 
-Click your package's overall score to see the Analysis page, which has
-suggestions for improving the package's score. Fix them, and release
-at least one new version every year to keep your maintenance score up.
+To determine if your package supports the latest versions of dependencies prior
+to publishing, run the `dart pub outdated` command (Dart SDK) or `flutter pub
+outdated` command (Flutter SDK).
 
-Pub site uses [pana][pana-url] to create maintenance suggestions.
-To get suggestions before publishing, run `pana` locally
-(using `--source path`), or validate your package against the
-[list of checks][pana-maintenance] manually.
+## Calculating pub points prior to publishing
 
-### Overall score
+The pub.dev site uses the analysis tool [pana][pana-url] to calculate pub
+points. This is run automatically on the pub.dev site whenever you publish a new
+package, or a new version of an existing package.
 
-The overall score is a weighted average of the individual scores:
-  - 50% popularity,
-  - 30% code health,
-  - 20% maintenance.
+You can view your *pub points report* to get suggestions before publishing, by
+running `pana` locally:
 
-You can find the overall score either near the top of the package's page
-or to the right of your package in any listing on this site.
+  1. When run `pana` will make modifications to the package, so start by making
+     a copy of the directory holding your package: `cp ~/dev/mypkg ~/tmp/mypkg`
 
-## Ranking
+  1. Make sure you have the latest pana tool: `dart pub global activate pana` (`pana`
+     changes frequently, so run this again frequenrly to update the pana tool)
 
-Default listings use composite scoring to sort packages. The score is
-based on the overall score, and if applicable, the platform specificity
-and the text match score is also factored in.
+  1. Run pana on the copy we made earlier: `dart pub global run pana --source path ~/tmp/mypkg`
```

Comment:
`pub global run pana ~/tmp/mypkg` (I think we no longer require `--source path` for local testing).

Reply:
Done